### ISSUE TITLE
"Next Document" vs "Next Document in List"

### ISSUE THREAD

**Issue by pablco** [adobe/brackets#14617]
### Prerequisites

* [ ] Can you reproduce the problem with `Debug -> Reload Without Extensions`?
* [ ] Did you perform a cursory search to see if your bug or enhancement is already reported?
* [ ] Did you read the [Troubleshooting guide](https://github.com/adobe/brackets/wiki/Troubleshooting)?

For more information on how to write a good bug report read [here](https://github.com/adobe/brackets/wiki/How-to-Report-an-Issue)
For more information on how to contribute read [here](https://github.com/adobe/brackets/blob/master/CONTRIBUTING.md)

### Description

"Next Document" & "Next Document in List" seem to have the same behavior

### Steps to Reproduce

1. Go to top menu
2. Click 'Navigate'
3. Click 'Next Document in List'

**Expected behavior:** [What you expected to happen]

I would expect that the next file in the main list will be open. 

Why? As a developer if I'm working in a specific folder it's expected that the file in the same folder have a working relationship, so if I'm in my Button folder I would have my Button.css and my Button.js file next to each other. So if I land using the quickfind into my Button.css I would expect that clicking/using the shortcut to move to the 'Next Document in List' I would open Button.js which is the file right next to it in the folder list

**Actual behavior:** [What actually happened]

The next Document, from my Open Documents list is open, which seems to be the same behavior as in the 'Next Document' command.

In summary both 'Next Document' & 'Next Document in List' they do the same while I would consider that the first command uses the open files and the later interacts with the main list.

### Versions

Please include the OS and what version of the OS you're running.
I'm running Mac Os Mojave but I can reproduce this in other versions of the OS

Please include the version of Brackets. You can find it under `Help -> About Brackets` (Windows and Linux) or `Brackets -> About Brackets` (macOS)
Release 1.13 Build

**Comment by shubhsnov** [comment]
This is not a bug, it is by design. Both NextDocument and NextDocumentInList work in the open files context. The only difference is that NextDocument works using MRU and NextDocumentInList just traverses the list in order.

**Comment by pablco** [comment]
Thanks for your response @shubhsnov — Are you able to reproduce the MRU behavior in the latest version? 

It does not work for me, even when I reload Brackets without extensions I am unable to reproduce it.
